### DISCUSSION TITLE
Adapting an existing type declarations file for DefinitelyTyped

### DISCUSSION THREAD

**Discussion by LoganDark** [DefinitelyTyped/DefinitelyTyped#56006] [Request a new @types package] [answered]
[`@atom/watcher`](https://github.com/atom/watcher/) lacks type declarations. I [made some](https://gist.github.com/LoganDark/7171a3c100b4fd9596920a15385b01cf). I want these to make their way into DefinitelyTyped, but I don't know how to contribute a new module.

Looking for someone else to generously do this for me and open a PR for it, since I'm on a big project and can't afford to spend any more time on these types (sorry!)

**Comment by jnv** [comment, marked as answer]
Hey @LoganDark, I will be happy to take care of it. I will add you as a main author and adjust the provided documentation to be proper docstrings.

Edit: PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56580

**Reply by LoganDark** [reply]
Thank you so much!
